components/support/nimbus-fml/src/backends/kotlin/gen_structs/imports.rs

```diff
@@ -28,9 +28,18 @@ impl<'a> ImportedModuleInitialization<'a> {
 }
 
 impl CodeDeclaration for ImportedModuleInitialization<'_> {
-    fn imports(&self, _oracle: &dyn CodeOracle) -> Option<Vec<String>> {
+    fn imports(&self, oracle: &dyn CodeOracle) -> Option<Vec<String>> {
         let p = self.inner.about().nimbus_package_name()?;
-        Some(vec![format!("{}.*", p)])
+        Some(
+            self.inner
+                .fm
+                .all_types()
+                .iter()
+                .filter_map(|t| oracle.find(t).imports(oracle))
+                .flatten()
+                .chain(vec![format!("{}.*", p)])
+                .collect::<Vec<_>>(),
+        )
```

Comment:
The workflow tests found an unrelated bug that the features that use the `Text` type causes a don't import all the needed type.

These problems are fixed in the `backends` crates.